ManifestData.cs

```diff
@@ -23,251 +31,332 @@ public class ManifestData {
-      data[220480][358470] = new Datum(0, 0, 0);
-
-      data[440323][322022] = new Datum(8229910699963260352, 2, 201589569);
-      data[429074][322022] = new Datum(6956365166520441230, 2, 201571015);
-      data[426014][322022] = new Datum(1538185595447097219, 2, 201570963);
-      data[424910][322022] = new Datum(1251221992794837269, 2, 201570650);
-      data[326589][322022] = new Datum(1999687630096794293, 3, 191280306);
-      data[301136][322022] = new Datum(2866879691166241287, 3, 191280316);
-      data[300763][322022] = new Datum(1944742430107480164, 3, 191280316);
-      data[293384][322022] = new Datum(6864442709661062902, 3, 191280316);
-      data[291145][322022] = new Datum(0330496884825252439, 3, 191280317);
-      data[284152][322022] = new Datum(6922743269332565255, 3, 191280316);
-      data[277544][322022] = new Datum(4959862740532303621, 3, 191280317);
-      data[269335][322022] = new Datum(2804553596758319623, 3, 191280313);
-      data[267252][322022] = new Datum(3417810446044054571, 3, 191280318);
-      data[264510][322022] = new Datum(5194706371718824589, 3, 191280314);
-      data[260924][322022] = new Datum(7503933396680891180, 3, 191280306);
-      data[258375][322022] = new Datum(9214799288230159850, 3, 191280304);
-      data[252786][322022] = new Datum(6635364740942276299, 3, 191084684);
-      data[250756][322022] = new Datum(6332807582508063459, 3, 191084684);
-      data[249913][322022] = new Datum(1518316276430466929, 3, 191084684);
-      data[249740][322022] = new Datum(5134711953944719494, 3, 191084684);
-      data[248828][322022] = new Datum(6408748820177783708, 3, 191084683);
-      data[248139][322022] = new Datum(5515868016021140366, 3, 191084683);
-      data[246379][322022] = new Datum(4423801769448698176, 2, 191045699);
-      data[244371][322022] = new Datum(0570457731767330233, 2, 191173592);
-      data[243520][322022] = new Datum(1221230661228297401, 4, 291798721);
-      data[226087][322022] = new Datum(7180495594989737482, 4, 275966134);
-      data[224995][322022] = new Datum(6536174967355028889, 3, 243487857);
-      data[224531][322022] = new Datum(7516537008051594086, 3, 243488546);
-      data[223249][322022] = new Datum(3867407920981508845, 3, 225475676);
-      data[222477][322022] = new Datum(8917339509592123029, 3, 225475410);
-      data[221394][322022] = new Datum(6888774931510513164, 3, 218106434);
-      data[220996][322022] = new Datum(7673940384630485679, 3, 211726262);
-      data[220675][322022] = new Datum(4383443687812477256, 3, 206909511);
-      data[220625][322022] = new Datum(8183524469048336858, 3, 206907989);
-      data[220480][322022] = new Datum(2824340380872620137, 3, 172228256);
+      return totalDownloadSize;
+    }
+
+    public static IEnumerable<SteamManifest> GetManifestsForVersion(int version, Package package = Package.All) {
+      return data[version].Where(m => package == Package.All || package == m.package);

```

Comment:
This is a low-performance implementation. If you know you need a second key, use a second key.
How about `Dictionary<int, Dictionary<Package, List<SteamManifest>> data`?
That way you can access via `data[version][package]`.